docs/.vuepress/components/rules-table.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <div v-if="kindMarks.length > 1" class="filter-tool">
+      Highlight:
+      <label v-for="kindMark in kindMarks" :key="kindMark">
+        <input type="checkbox" :value="kindMark" v-model="checkedKindMarks" />
+        <span class="emoji">{{ kindMark }}</span>
+      </label>
+    </div>
+    <div class="table-root" ref="tableRoot">
+      <slot />
+    </div>
+  </div>
+</template>
+
+<script>
+const emojis = ['3️⃣', '2️⃣', '⚠️']
```

Comment:
`Intl.Segmenter` is required to properly split these emojis, but FireFox doesn't support it, so we'll handle it ourselves.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter